app/controllers/green_lanes/applicable_exemptions_controller.rb

```diff
@@ -7,24 +7,58 @@ class ApplicableExemptionsController < ApplicationController
                   :disable_search_form
 
     def new
-      @exemptions_form = exemptions_form
+      @exemptions_form = build_exemptions_form
+      @back_link_path = determine_back_link_path
 
       render_exemptions_questions
     end
 
     def create
-      @exemptions_form = exemptions_form
+      @exemptions_form = build_exemptions_form
 
       if @exemptions_form.valid?
         next_page = determine_next_page
         redirect_to handle_next_page(next_page)
       else
+        @back_link_path = determine_back_link_path
+
         render_exemptions_questions
       end
     end
 
     private
 
+    def determine_back_link_path
+      permitted_params = params.permit(:commodity_code, :country_of_origin, :moving_date, :c1ex, :c2ex, ans: {})
+
+      if category == 2
+        new_green_lanes_applicable_exemptions_path(
+          category: 1,
+          commodity_code: permitted_params[:commodity_code],
+          country_of_origin: permitted_params[:country_of_origin],
+          moving_date: permitted_params[:moving_date],
+          ans: parsed_ans(permitted_params[:ans]),
+          c1ex: permitted_params[:c1ex],
+        )
+      else
```

Comment:
to understand: what is this case?
I guess  category == 1,
maybe we can add a comment

Reply:
So this is because its used for the cat 1 and 2 questions. the bottom case is if you are in the cat1 questions and click back so you go back to the good details page.

the top one is if you are on cat 2 questions then you go back to the cat 1 questions